install-webcam.sh

```diff
@@ -0,0 +1,44 @@
+apt-get install ffmpeg
+
+echo "Cloning GSPCA Repo..."
+git clone https://github.com/grandchild/gspca-kinect2.git && cd gspca-kinect2
+echo "Building & Installing Modules (copying into /lib/modules/`uname -r`/kernel/drivers/kinect)"
+make -C /lib/modules/`uname -r`/build  M=`pwd` SRCROOT=`pwd` clean modules
+cp gspca_main.ko gspca_kinect_main.ko
+cp gspca_kinect_main.ko /lib/modules/`uname -r`/kernel/drivers/kinect/gspca_kinect_main.ko
+cp gspca_kinect2.ko /lib/modules/`uname -r`/kernel/drivers/kinect/gspca_kinect2.ko
```

Comment:
Wouldn't these have to be run as root? Hint: The answer is _not_ to run the whole script as root.